ext/hivemq-edge-openapi-2024.8-SNAPSHOT.yaml

```diff
@@ -131,14 +131,8 @@ paths:
         "401":
           content:
             application/json:
-              examples:
-                authentication-invalid:
-                  description: Unable to authenticate credentials
-                  summary: The requested credentials could not be authenticated
-                  value:
-                    title: Invalid username and/or password
               schema:
-                $ref: '#/components/schemas/ApiErrorMessage'
+                $ref: '#/components/schemas/Errors'
```

Comment:
Perhaps we can to give the Errors class a nice naming now. However we should discuss with Nicolas